src/components/organisms/HelpCenter/HelpCenter.scss

```diff
@@ -1,6 +1,18 @@
 @import "scss/constants.scss";
 
 .HelpCenter {
+  &__iframeContainer {
+    position: relative;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__backButton {
+    position: absolute;
+    top: 90px;
+    left: 25px;
```

Comment:
Is it worth keeping these measures in some vars?

Reply:
@yurii-lubynets fixed in 5fe10535b862a45805d69a34bcfcbfe5e1666ef1

Reply:
Those constants seem arbitrary. It is always nice if they're based on existing layout references. 
Not  examples of what constants should be used here,
but a reminder values in  `HelpCenter.scss` can be based on existing ones and/or be some integer scale of them.

- Here is how some calculations were done (for the chat/help frame):

```scss
 &__private-chat {
    margin-top: $header-height + 2 * $spacing--md;
  }

  &__help-center-icon {
    margin-top: 2 * $header-height + 3 * $spacing--md;
  }

  &__create-icon {
    margin-top: 3 * $header-height + 4 * $spacing--md;
  }
```

- There are also some useful in `constants.scss' like:
```scss
$navbar-height: 66px;
$sidebar-schedule-width: 40%;
$footer-height: 30px;
$reserved-page-height: $navbar-height + $footer-height;
$chat-sidebar-width: 360px;
```